pb_app/lib/workflows.dart

```diff
@@ -2,15 +2,16 @@ import 'package:flutter/widgets.dart';
 import 'package:pb_app/config.dart';
 import 'package:pb_app/modals.dart';
 import 'package:pb_app/secrets.dart';
+import 'package:pb_app/submission_form.dart';
 import 'package:pocketbase/pocketbase.dart';
 
 final client = PocketBase(Config.baseUrl);
 
 class Workflows {
-  static Future<UserAuth> authenticate() {
+  static Future<UserAuth> authenticate({email, password}) {
     return client.users.authViaEmail(
-      Secrets.email,
-      Secrets.password,
+      email ?? Secrets.email,
+      password ?? Secrets.password,
     );
   }
```

Comment:
Ah, nice find. Here's my suggestion: delete this authenticate workflow and create an extension on `client.users` called `authViaSecrets` with no arguments and use that instead